subprojects/ide-provisioning/src/test/groovy/org/gradle/profiler/TeeOutputStream.java

```diff
@@ -0,0 +1,52 @@
+package org.gradle.profiler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.List;
+
+public class TeeOutputStream extends OutputStream {
```

Comment:
This was copy/pasted from `gradle-profiler`, but I think we should create some project with test fixtures or so. WDYT?

Reply:
`TeeOutputStream` is a part of `common-io`, that was already included to `gradle-profiler`. So decided to remove own implementations and use common one